src/Components/Facility/AssetCreate.tsx

```diff
@@ -589,17 +590,38 @@ const AssetCreate = (props: AssetProps) => {
                 />
               </div>
             </div>
-            <Button
-              id="asset-create"
-              color="primary"
-              variant="contained"
-              type="submit"
-              style={{ marginLeft: "auto" }}
-              onClick={(e) => handleSubmit(e)}
-              startIcon={<CheckCircleOutlineIcon>save</CheckCircleOutlineIcon>}
-            >
-              {assetId ? "Update" : "Create"}
-            </Button>
+            <div className="flex justify-center sm:justify-start gap-x-4 gap-y-2 flex-wrap">
+              <Button
+                id="asset-create"
+                color="primary"
+                variant="contained"
+                type="submit"
+                onClick={(e) => handleSubmit(e)}
+                startIcon={
+                  <CheckCircleOutlineIcon>save</CheckCircleOutlineIcon>
+                }
+                className="w-full sm:w-auto"
+              >
+                {assetId ? "Update" : "Create"}
+              </Button>
+              <Button
+                id="asset-cancel"
+                color="primary"
+                variant="outlined"
+                type="button"
+                onClick={() =>
+                  navigate(
+                    `${
+                      assetId ? `/assets/${assetId}` : `/facility/${facilityId}`
+                    }`
+                  )
```

Comment:
```suggestion
                  navigate(
                      assetId ? `/assets/${assetId}` : `/facility/${facilityId}`
                  )
```

I don't think we need the outer string interpolation 🤔

Reply:
Yeah, the return type from ``` assetId ? `/assets/${assetId}` : `/facility/${facilityId}` ``` is a string. I thought it might be some JSX. Thanks @khavinshankar